src/components/cell.tsx

```diff
@@ -1,11 +1,11 @@
-import { CellOptions } from "../types/types";
+  import { CellOptions } from "../types/types";
 
-interface Props {
-  type: CellOptions;
-}
+  interface Props {
+    type: CellOptions;
+  }
 
-function Cell({ type }: Props) {
-  return <div className={`cell ${type}`} />;
-}
+  function Cell({ type }: Props) {
```

Comment:
Object destructuring is especially unsafe in function parameters. If the caller sends in `null` here, your app will crash in a way that’s *very* hard to troubleshoot. Suggest *never* using object destructuring in function parameters (even though it’s commonly done because folks don’t recognize the risk)